monitoring/frontend.go

```diff
@@ -387,6 +387,16 @@ func Frontend() *Container {
 							PanelOptions:      PanelOptions().LegendFormat("{{category}}"),
 							PossibleSolutions: "none",
 						},
+						{
+							Name:              "90th_percentile_updatecheck_requests",
+							Description:       "90th percentile successful update requests",
```

Comment:
```suggestion
							Description:       "90th percentile successful update-check requests (sourcegraph.com only)",
```

because no private instance will be serving update-check requests